vcf_annotation_tools/vcf_readcount_annotator.py

```diff
@@ -33,6 +34,12 @@ def define_parser():
         help="Path to write the output VCF file. If not provided, the output VCF file will be "
             +"written next to the input VCF file with a .readcount.vcf file ending."
     )
+    parser.add_argument(
+        "-z", "--write-missing-counts-as-zero",
+        help="The readcount fields for any positions that are not found in the bam_readcount_file(s) "
+            +"will be filled with 0s.",
```

Comment:
Suggested change in help text:
"assume that positions not found in the bam-readcount file(s) have zero depth.  (This is the default behavior of bam-readcount) These positions will have all their readcount fields set to 0".

Reply:
Right now this is not the default. If we want this behavior to be the default, do we even need this flag?

Reply:
Yeah I think this should just happen by default. Not sure when we would want to turn that behavior off. I guess if someone knows they are using an incomplete bam-readcount result, they could turn off this default and get NA’s for the missing values instead. Not sure if it is worth supporting that use case?